cdap-ui/app/cdap/components/CardActionFeedback/index.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+  Usage:
+    <CardActionFeedback
+      type='success | warning | danger | loading'
+      message='some feedback message'
+      stackTrace='stack trace message that will get rendered in <pre>'
+    />
+*/
+import React, {Component, PropTypes} from 'react';
+require('./CardActionFeedback.less');
+
+var classNames = require('classnames');
+
+export default class CardActionFeedback extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isExpanded: false
+    };
+  }
+
+  _getIcon() {
+    let icon = classNames('fa',
+      { 'fa-check': this.props.type === 'SUCCESS' },
+      { 'fa-exclamation': this.props.type === 'DANGER' },
+      { 'fa-exclamation-triangle': this.props.type === 'WARNING' },
+      { 'fa-spinner fa-spin': this.props.type === 'LOADING' }
+    );
+
+    return <span className="feedback-icon"><span className={icon}></span></span>;
+  }
+
+  _getStackTrace() {
+    if (this.props.stackTrace) {
```

Comment:
we should not call it StackTrace, it's extendedMessage. Also, what happens when there is not extendedMessage. Is there a down arrow shown ? Should it be ?